src/scancode/cli.py

```diff
@@ -34,6 +34,7 @@
 from collections import OrderedDict
 from functools import partial
 import os
+import datetime
```

Comment:
we like imports to be grouped, then sorted. This should go right after the collections import